sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc

```diff
@@ -626,75 +596,73 @@ static_assert(std::is_same_v<t3, bool>);
 Properties should provide semantically meaningful functions to access the parameters' values.
 ====
 
-For functions that take a property list parameter, it is possible to restrict
+For functions that take a `properties` as a parameter, it is possible to restrict
 the parameter to require a specific property.  If the property takes a
 compile-time constant parameter, it is further possible to restrict the
 function to take a property list that has a property with a specific parameter.
 The following example demonstrates this, where `foo` is runtime property and
 `bar` is a compile-time constant property.
 
 ```c++
-template<typename propertyListT>
-std::enable_if_t<is_property_list_v<propertyListT>> my_func1(propertyListT p);
+template<typename propertiesT>
```

Comment:
Nit: `PropertiesT` would be more consistent, but again I don't want to block this PR.